README.rst

```diff
@@ -10,6 +10,25 @@ Overview
 
 This app enables reuse of username and email after user retirement on Open edX Platfrom
 
+Currently, once a user is retired, their email and username is not allowed to be reused for registration for new user.
+However, there are some use-cases where it is useful to allow this reuse, such as, if the user management is handled by an external system, which manages the user in LMS through API calls.
+
+Further the retired username is changed from a hash value to a friendly name of the form ``deleted_user_3``.
+This is done so as the new format looks far friendlier than the orginal hashed username in places like the discussion forum.
+
+Installation
+------------
+
+1. Login to LMS Shell
+2. Checkout this repository
+3. ``cd enable-retired-username-email``
+4. ``pip install -e .``
+5. This app could also be installed using `ansible-configurations <https://github.com/openedx/configuration/blob/f676c356a5424a52ebff01da7a8a7d96189f2579/playbooks/roles/edxapp/defaults/main.yml#L542>`_
+
+No other configuration needed to run the app
```

Comment:
This is with the addition to ``Retirement Service`` so we need to enable Retirement Service in order to use this app fully.

Reply:
AFAIK, the retirement service itself is enabled by default.
We however do need to configure the retirement states, which I have added to the docs.

I feel like instructions to enable the retirement cron job is beyond the scope of this docs, as the user might not want to use the cron job but instead choose to trigger the retirement workflow using APIs.